src/main/java/com/baidu/hugegraph/structure/schema/IndexLabel.java

```diff
@@ -273,6 +273,7 @@ public IndexLabelV56(IndexLabel indexLabel) {
             super(indexLabel.name);
             E.checkArgument(indexLabel.rebuild,
                             "The rebuild of indexlabel must be true");
+            this.id = indexLabel.id();
```

Comment:
bump the version?

Reply:
no need right now. This bug only affects the situation that using hugegraph-tools 1.5 migrate/restore indexlabels to HugeGraphServer 0.10.